microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -250,7 +250,27 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
+## Use settings for Allocation sampling to reduce garbage
 
+By default, the **Allocation sampling** profiling type only reports allocations that are still alive at the end of the recording session.  Objects that are generated, garbage collected (GC'd), and then disappear aren't tracked by either **Allocation sampling** or **Allocation instrumentation on timeline**.
+
+You can configure the **Allocation sampling** profiling type to track objects that are discarded by GC with settings.
+
+1. Click the **Allocation sampling** radio button.
+
+1. Click the **Include objects discarded by major GC** and **Include objects discarded by minor GC** settings.
+
+    ![Allocation sampling GC settings.](../media/memory-problems-memory-allocation-sampling-gc-settings.png)
```

Comment:
```suggestion
   ![Allocation sampling GC settings.](../media/memory-problems-memory-allocation-sampling-gc-settings.png)
```
make sure indents align, within any Markdown list.  Sometimes all items are pushed to the right by 1 space, when you apply 4-char indent per some linter conventions.  Can break indent or numbering if not uniform in a given list.